lnbits/core/templates/empty_public.html

```diff
@@ -0,0 +1,3 @@
+{% extends "public.html" %} {% from "macros.jinja" import window_vars with
+context %} {% block page %} {% endblock %} {% block scripts %} {{
+window_vars(user) }} {% endblock %}
```

Comment:
```suggestion
 }} {% endblock %}
```

why does it need `window_vars(user)` if its public?

Reply:
open issue